cmd/security-profiles-operator/main.go

```diff
@@ -225,3 +233,48 @@ func runDaemon(ctx *cli.Context) error {
 	setupLog.Info("ending daemon")
 	return nil
 }
+
+func runBindingWebhook(ctx *cli.Context) error {
```

Comment:
Why not run the webhook with the main deployment? as part of the `manager` subcommand. This way it would all be part of the same pod.